archive/compression/testdata/test.json

```diff
@@ -0,0 +1,5902 @@
+[
```

Comment:
Can we move test data to gist or somewhere else? (could be downloaded from TestMain())

Reply:
Done